src/elements/common/empty-view/EmptyView.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import { EmptyState, Text } from '@box/blueprint-web';
+import { Files, FolderFloat, HatWand, OpenBook } from '@box/blueprint-web-assets/illustrations/Medium';
+
+import { VIEW_ERROR, VIEW_FOLDER, VIEW_METADATA, VIEW_SEARCH, VIEW_SELECTED } from '../../../constants';
+
+import messages from './messages';
+
+import type { View } from '../../../common/types/core';
+
+export interface EmptyViewProps {
+    isLoading: boolean;
+    view: View;
+}
+
+const EmptyView = ({ isLoading, view }: EmptyViewProps) => {
```

Comment:
I renamed the previous `EmptyState` to `EmptyView` so not to be confused with the blueprint `EmptyState`